bin/deploy

```diff
@@ -455,7 +455,9 @@ main() {
     )
   fi
 
-  echo "Done. Run the following:"
+  echo ""
+  echo "::: Deploy completed."
```

Comment:
I don't really understand what the `::: ` is for.

Reply:
I tried to follow what was done above, but probably I should have not changed other's print outs.